sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOTenantRenameSitePrioritization.md

```diff
@@ -0,0 +1,56 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/remove-spotenantrenamesiteprioritization
+applicable: SharePoint Online
+title: Remove-SPOTenantRenameSitePrioritization
+author: AnfraMsft
+ms.author: anfra
+ms.reviewer: anushmag
+manager: anushmag
+schema: 2.0.0
+ms.topic: reference
+---
+
+# Remove-SPOTenantRenameSitePrioritization
+## SYNOPSIS
+Allows you to remove the prioritization previously assigned to a site for early execution, as part of Advanced Tenant Rename.
```

Comment:
>Advanced Tenant Rename [](http://example.com/codeflow?start=102&length=22)

do you want to inline href this like you did for Set-SPOTenantRenameSitePrioritization?